frontend/src/components/CreateCsv.jsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "antd";
+
+const CreateCsv = ({ tableData, selectedSalaryDate, finnishDateNoDots }) => {
```

Comment:
To make this component reusable it would be better to pass the filename as a prop instead of dates and then creating a filename based on it. This currently only creates csv files that start with `palkat_`. If it was used in some other report it would make more sense to use some other name.